sympy/solvers/diophantine/diophantine.py

```diff
@@ -598,7 +621,10 @@ def solve(self, parameters=None, limit=None) -> DiophantineSolutionSet:
                             result.add((x_0, y_0))
 
             else:
-                s = BinaryQuadratic(self.equation, free_symbols=var[::-1]).solve(parameters=[t, u])  # Interchange x and y
+                s = BinaryQuadratic(self.equation,
+                                    free_symbols=var[::-1]).solve(
+                                        parameters=[t, u]
+                                    )  # Interchange x and y
```

Comment:
I guess this is the kind of situation where an intermediate variable would help.

Reply:
Last time (above) it didn't break on the single argument. I guess it did now because of line length.

Reply:
> I guess this is the kind of situation where an intermediate variable would help.

True, although same as https://github.com/sympy/sympy/pull/22437#discussion_r744184161, I don't really get why the formatting looks like this. But `split_before_expression_after_opening_paren = True` (d864ba227c) improves it:

```python
                s = BinaryQuadratic(
                    self.equation, free_symbols=var[::-1]
                ).solve(parameters=[t, u])  # Interchange x and y
```

> Last time (above) it didn't break on the single argument. I guess it did now because of line length.

With black? Could be; black wraps at 88 by default.